modules/shortcodes/vr.php

```diff
@@ -129,3 +128,29 @@ function jetpack_vr_viewer_shortcode( $atts ) {
 }
 
 add_shortcode( 'vr', 'jetpack_vr_viewer_shortcode' );
+
+// Gutenberg!
+add_action( 'admin_init', 'jetpack_register_block_type_vr' );
+function jetpack_register_block_type_vr() {
+	if ( ! function_exists( 'register_block_type' ) ) {
+		return;
+	}
+
+	wp_register_script(
+		'jetpack_vr_viewer_shortcode_editor_script',
+		Jetpack::get_file_url_for_environment( '_inc/build/shortcodes/js/blocks/vr-block.min.js', 'modules/shortcodes/js/blocks/vr-block.js' ),
+		array( 'wp-blocks', 'wp-element' )
```

Comment:
You are using also `wp-i18n` in your script.

Reply:
Fixed, thanks!